### DISCUSSION TITLE
Help building a .jar?

### DISCUSSION THREAD

**Discussion by Ethonodon** [community/community#127615] [Programming Help]
### Select Topic Area

Question

### Body

I am completely new to github and coding, and I'm trying to build a .jar out of the 1.18.2 branch of https://github.com/Ethonodon/FossilsArcheologyRevival. I have googled like crazy to search for ways to build a jar, and the few methods I've been able to understand have not worked. When I try and build it with commandprompt, it gives me an error. When I try and build it with Eclipse IDE, it won't recognize the files as a "project" despite them being imported and functioning within the application. Supposedly I can also build it directly in github, but I have no idea where I'd do that. I'm confused- could there maybe be some error on my computer's end, am I doing something wrong, or is there something wrong with the repository itself? Any help would be appreciated. Thanks!

**Comment by wis** [comment]
This seems general programming related, and not related to GitHub the site/platform/product. 
If you want general programming related help you should post your question on StackOverflow.

**Reply by Ethonodon** [reply]
what do you mean this isn't github related? I'm asking about an issue building a jar from a github file and one of the methods I've heard about uses github specifically

**Reply by queenofcorgis** [reply]
Hi @Ethonodon 👋🏼 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.  The Repositories category is for conversations specific to that feature, while the Programming Help category can assist with more building and project related questions. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information. 

Congrats on joining GitHub and starting your coding journey ⭐

**Comment by Chari-Z** [comment]
Hey @Ethonodon 

Looks like it's a Gradle project. You'll need to use gradle to build it. Have you looked into that at all? You really only need to install Gradle and do `gradle build` in the a Terminal based in the folder of all the files.

**Reply by Ethonodon** [reply]
How do I know what gradle version to install?
